CONTRIBUTING.md

```diff
@@ -69,6 +69,18 @@ If the bytecode is the same, but query results are different, then the bug is so
 The `testing/test.all` is a starting point for adding functional tests using a similar syntax to SQLite.
 The purpose of these tests is to verify behavior matches with SQLite and Limbo.
 
+### Prerequisites
+
+1. [Cargo-c](https://github.com/lu-zero/cargo-c) is needed for building C-ABI compatible library. You can get it via:
+```console
+cargo install cargo-c
+```
+2. [SQLite](https://www.sqlite.org/index.html) is needed for compatibility checking. You can install it using `brew`:
+```console
+brew install sqlite
```

Comment:
maybe add install instructions for some other platforms too, or just "you can install it e.g. on Mac using  `brew`:

Reply:
Homebrew is also available for linux, but indeed it is not so mainstream as on Mac.
(for me it was alos the easiest way to install the recent version on linux)

Anyway I agree that it would be best to add Windows specific instructions. 
I see that PR is already merged by penberg I will add a new one with `choco` based instructions for Windows